src/Views/TryInstallView.vala

```diff
@@ -78,13 +78,13 @@ public class Installer.TryInstallView : AbstractInstallerView {
         );
 
         var clean_install_button = new InstallTypeButton (
-            _("Do a Clean Install"),
+            _("Erase Disk and Install"),
             "edit-clear",
             _("Erase everything and install a fresh copy of %s.").printf (Utils.get_pretty_name ())
         );
 
         var custom_button = new InstallTypeButton (
-            _("Custom (Advanced)"),
+            _("Insall with custom partitioning"),
```

Comment:
Small typo, and it looks like before we were using Title Case here
```suggestion
            _("Install with Custom Partitioning"),
```

Reply:
was fixed with https://github.com/elementary/installer/pull/510/commits/ac65c2533590b18c030779fbf20313423ec821cc